homework-submission/todo_react/src/Info.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import logo from './logo.svg';
+
+function SomeInfo(props) {
+
+    return <div className='topic'>
+        {props.img}
+        {props.topic}
+
+
+    </div>
+};
+
+class Info extends React.Component {
+    state = {
+        some_info: [{
```

Comment:
`some_info` name didn't made any sense for me, it would be great if you picked another name, plus you have another function in this file has the same name, which is confusing in some point. 😞